backend/schema/validate.go

```diff
@@ -464,7 +466,19 @@ func validateVerbMetadata(scopes Scopes, n *Verb) (merr []error) {
 				case *IngressPathLiteral:
 				}
 			}
+		case *MetadataCronJob:
+			if hasCron {
+				merr = append(merr, errorf(md, "only a single cron schedule is allowed per verb"))
+				continue
+			}
+			hasCron = true
 
+			schedule, err := cron.Parse(md.Cron)
+			if err != nil {
+				merr = append(merr, err)
+			} else if err := cron.Validate(schedule); err != nil {
```

Comment:
I think it would be preferable for `cron.Parse()` to call `Validate()` internally, so you can never have an invalid schedule. What's the advantage of it being separate?